shipment_advice_planner_toursolver/models/toursolver_backend.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import datetime, timedelta
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class TourSolverBackend(models.Model):
+    _name = "toursolver.backend"
+    _description = "TourSolver Backend"
+
+    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
+    resource_properties_definition = fields.PropertiesDefinition(
+        string="Resource Properties"
+    )
+    url = fields.Char()
+    api_key = fields.Char()
+    delivery_window_disabled = fields.Boolean()
+    delivery_window_start = fields.Float(default=8.0)
```

Comment:
These delivery_window_start and delivery_window_end fields are default values if none is provided on the partner. This should be reflected into the the field name and in a help text.